src/map/map.cpp

```diff
@@ -178,6 +178,12 @@ int32 do_init(int32 argc, char** argv)
         }
     }
 
+    if (!std::filesystem:exists("./scripts/settings/main.lua"))
```

Comment:
@zach2good My last commit
a) probably isn't the right thing
and
b) doesn't have any kind of delay so that our slower users who never look at the log after "a crash" will see it.

Reply:
I think i have something for this, I'll take a look tomorrow

Reply:
> I think i have something for this, I'll take a look tomorrow

_nudge_

Reply:
I... do not remember what I would have for this... `std::filesystem:exists` is probably good.